packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/SupportCta.tsx

```diff
@@ -1,20 +1,14 @@
 import React from 'react';
 import { addRegionIdAndTrackingParamsToSupportUrl } from '@sdc/shared/dist/lib';
 import { Tracking } from '@sdc/shared/dist/types';
-import { brandAlt, neutral, space } from '@guardian/src-foundations';
+import { space } from '@guardian/src-foundations';
 import { css, SerializedStyles } from '@emotion/react';
 import { Hide } from '@guardian/src-layout';
 import { Button } from './Button';
 import { ChoiceCardSelection } from '../ChoiceCardsButtonsBanner';
 
 const buttonOverrides = css`
     margin-right: ${space[3]}px;
-    background: ${brandAlt[400]};
-    color: ${neutral[0]} !important;
-
-    &:hover {
-        background-color: ${brandAlt[200]};
-    }
```

Comment:
I may have expected this change to affect the bespoke choice cards banner but it looks good locally! Although I think these changes would have a knock on effect to any existing moment template banners / stories using that choice cards component.

Reply:
There are a couple of stories it would be good to update but I believe the Europe banner is a moment template banner with choice cards? Which may need the `primaryCta` settings updated after this goes in.

Reply:
Yep good spot, thanks, have updated all moment choice card banners PrimaryCTA's now.